develocity-maven.yml

```diff
@@ -80,22 +96,26 @@ spec:
     fi
     if [[ ("${appliedCustomDv}" = "true" || "${appliedCustomCcud}" = "true") && "${enforceUrl}" = 'true' ]]
     then
-      mavenOpts="${mavenOpts} -Dgradle.enterprise.url=${url}"
+      mavenOpts="${mavenOpts} -Dgradle.enterprise.url=${url} -Ddevelocity.url=${url}"
     elif [[ "${appliedCustomDv}" = "false" && "${appliedCustomCcud}" = "false" ]]
     then
-      mavenOpts="${mavenOpts} -Dgradle.enterprise.url=${url}"
+      mavenOpts="${mavenOpts} -Dgradle.enterprise.url=${url} -Ddevelocity.url=${url}"
     fi
     local existingMavenOpts=$(if [ ! -z "$MAVEN_OPTS" ]; then echo "${MAVEN_OPTS} "; else echo ''; fi)
     export MAVEN_OPTS=${existingMavenOpts}${mavenOpts}
   }
-
-  function dvCoordinates() {
-    local coordinates='com.gradle:gradle-enterprise-maven-extension'
+  
+  function detectDvExtension() {
     if [ ! -z "${customMavenExtensionCoordinates}" ]
     then
-      coordinates="${customMavenExtensionCoordinates}"
+      echo "$(detectExtension $1 $customMavenExtensionCoordinates)"
+    else
+      local appliedDefaultExtension="$(detectExtension $1 'com.gradle:gradle-enterprise-maven-extension')"
+      if [ "${appliedDefaultExtension}" = "false" ]; then
+        appliedDefaultExtension="$(detectExtension $1 'com.gradle:develocity-maven-extension')"
+      fi
+      echo "${appliedDefaultExtension}"
     fi
-    echo "${coordinates}"
   }
 
   function ccudCoordinates() {
```

Comment:
I just saw a bug on the line below I'll fix in a subsequent PR (wrong default CCUD coordinates)